borg/testsuite/upgrader.py

```diff
@@ -77,7 +79,7 @@ def test_convert_segments(tmpdir, attic_repo):
     """
     # check should fail because of magic number
     assert not repo_valid(tmpdir)
-    print("opening attic repository with borg and converting")
+    logger.info("opening attic repository with borg and converting")
```

Comment:
if the logger is setup properly, that would as well.. :) but i can revert that as well if you prefer...